spec/controllers/api/personal_logs_controller_spec.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe Api::PersonalLogsController do
+  include ControllerHelpers
+
+  let(:bob) { users(:bob) }
+  let(:alice) { users(:alice) }
+  let(:api_user) { bob.api_users.create }
+  let(:credentials1) { encryptables(:credentials1) }
+
+  context 'GET index' do
+    it 'returns right amount of logs' do
+      login_as(:alice)
+      PaperTrail.request(whodunnit: alice.id) do
+        credentials1.touch
```

Comment:
I mean, this works and is fine.
Just for readabilities' sake use `save_with_version` instead of `.touch` as we use `save_with_version` everywhere else.

Reply:
This should be done now :)